Sources/Networking/HTTPClient/HTTPRequest.swift

```diff
@@ -119,6 +119,24 @@ extension HTTPRequest.Path {
         }
     }
 
+    /// Whether requests to this path can be cached using `ETagManager`
+    var isCached: Bool {
+        switch self {
+        case .getCustomerInfo,
+                .getOfferings,
+                .getIntroEligibility,
+                .logIn,
+                .postAttributionData,
+                .postOfferForSigning,
+                .postReceiptData,
+                .postSubscriberAttributes,
+                .postAdServicesToken:
+            return true
+        case .health:
+            return false
```

Comment:
nice! 
absolute nitpick, FFTI: I would name this `shouldCache` or something like that, to reflect that it's not something that _has_ already happened, rather, whether they should be. 
Even further, this isn't entirely a cache either - eTags are just about saving up some kilobytes of HTTP traffic, but it's not a traditional cache in that it doesn't have a specific TTL or anything.
So maybe something like `shouldSendEtag` would be more representative of that

Reply:
> I would name this shouldCache or something like that, to reflect that it's not something that has already happened, rather, whether they should be.

Great point!

> `shouldSendEtag`

💯